hadoop-crypto/src/main/java/com/palantir/crypto2/hadoop/StandaloneEncryptedFileSystem.java

```diff
@@ -135,7 +137,11 @@ public boolean exists(Path path) throws IOException {
     public boolean delete(Path path, boolean recursive) throws IOException {
         // EncryptedFileSystem does not handle recursive deletes because it needs to rely on the KeyStorageStrategy.
```

Comment:
took me a minute to understand this comment without prior context, so maybe worth explaining a bit more thoroughly: `In contrast to EncryptedFileSystem (which works with arbitrary KeyStorageStrategies), a StandaloneEncryptedFileSystem always uses a FileKeyStorageStrategy which stores key materials as files adjacent to payload files. This allows us to implement recursive deletion of folders (and rename, see below) by simply renaming the folder in the underlying delegate file system: it deletes (or renames) both the payload files and the adjacent key material files. When deleting files, we need to explicitly delete the payload file and the key material file.`

Reply:
Gonna think about this...seems like something is a bit off if we need so much explanation for a single method override but maybe this is ok.